README.md

```diff
@@ -1,33 +1,62 @@
-# ETH Tech Tree Challenge Grader
+# ETH Tech Tree Backend
 
 This express server application is used to grade submitted challenges from the ETH Tech Tree
 
+## Requirements
+
+Before you begin, you need to install the following tools:
+
+- [Node (>= v18.17)](https://nodejs.org/en/download/)
+- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
+- [Git](https://git-scm.com/downloads)
+
 ## Getting Started
 
-1. Download repo & install dependencies
+1. Clone this repo & install dependencies
+
+    ```
+    git clone https://github.com/BuidlGuidl/eth-tech-tree-backend.git
+    cd eth-tech-tree-backend
+    yarn install
+    ```
+
+2. Set up your environment variables (and optionally, a local Firebase instance):
+   Make a copy of the `.env.example` file and name it `.env.local` and fill in the required environment variables.
+
+    You will need to set up an [Etherscan API Key](https://etherscan.io/apis).
+
+    (Optional) Start the firebase emulators (vs set up a live Firebase instance). You will need to install the [firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli) `npm i -g firebase-tools` and run the following command:
+
+    ```bash
+    # You might need to add a real "--project <projectName>" (run firebase projects:list)
+    firebase emulators:start
+    ```
+
+3. Seed data in your local Firebase instance:
+
+    Copy the `packages/local_db/seed.sample.json` to `packages/local_db/seed.json` and tweak the data as you see fit. Then run the following command:
+
+    ```bash
+    yarn seed
+    ```
 
-```
-git clone https://github.com/BuidlGuidl/eth-tech-tree-backend.git
-cd eth-tech-tree-backend
-yarn install
-```
+    To seed it to empty _*live*_ firestore instance you can use `yarn seed --force-prod`. If there is data in the live instance, it will not seed it again to bypass it use `yarn seed --reset --force-prod`
```

Comment:
You might consider separating the local firebase setup instructions from the live firebase instructions. I think focusing on local may be better choice for the "Getting Started" section as you onboard more devs to help out with backend